### ISSUE TITLE
yAxis label not center aligned vertically

### ISSUE THREAD

**Issue by Gunner17** [recharts/recharts#2105]
Please see this example, the "insideLeft" label does not seem to be vertically center aligned. When I have text with dynamic widths, what is the best way to center align the yAxis "insideLeft" label?
https://recharts.org/en-US/examples/ComposedChartWithAxisLabels

<img width="596" alt="Screen Shot 2020-04-23 at 3 04 07 AM" src="https://user-images.githubusercontent.com/11354728/80087051-6e509580-850f-11ea-832c-4b1964495a38.png">

**Comment by xile611** [comment]
Please try the latest version.

**Comment by charleswduk** [comment]
This is an issue for me even using the latest version. @Gunner17 Did you find a solution please?

**Comment by cmauersberger** [comment]
The latest version did not solve it either for me, @xile611 .
My solution @charleswduk / @Gunner17 : The problem is, that the "text-anchor" css property is not set to "middle" (but "start", so the label starts in the middle). The label seems to ignore the "text-anchor" property if I passed it manually via the label options. I solved this by adding a className (this property is not ignored, but added to the text tag) and setting "text-anchor: middle" to the css class.

For the x axis, "text-anchor: middle" seems to be the standard.
